### PR TITLE
allow multi-user / multi-org to be set on each org

### PR DESCRIPTION
**Pull request by nicpottier**
Also removes import flag since we always have that set to 0 anyways and not allowing imports causes more troubles than it solves.

### CODE REVIEW

File: temba/contacts/tests.py [hunk 1]
```diff
@@ -3477,7 +3477,7 @@ def test_update_and_list(self):
         response = self.client.get(reverse("contacts.contact_update", args=[self.joe.id]))
 
         self.assertEqual(
-            list(response.context["form"].fields.keys()), ["name", "groups", "urn__twitter__0", "urn__tel__1", "loc"],
+            list(response.context["form"].fields.keys()), ["name", "groups", "urn__twitter__0", "urn__tel__1", "loc"]
```
Comment (nicpottier) [inline temba/contacts/tests.py:3480]: Not sure why this is changing.. my black seems to be the same version.

File: temba/orgs/models.py [hunk 2]
```diff
@@ -257,9 +265,7 @@ def get_unique_slug(cls, name):
             return unique_slug
 
     def create_sub_org(self, name, timezone=None, created_by=None):
-
-        if self.is_multi_org_tier() and not self.parent:
```
Comment (nicpottier) [inline temba/orgs/models.py:261]: We didn't let nesting of child orgs, we have clients that specifically want this so it isn't excluded here.

File: temba/orgs/models.py [hunk 3]
```diff
@@ -277,6 +283,8 @@ def create_sub_org(self, name, timezone=None, created_by=None):
                 slug=slug,
                 created_by=created_by,
                 modified_by=created_by,
+                is_multi_user=self.is_multi_user,
+                is_multi_org=self.is_multi_org,
```
Comment (nicpottier) [inline temba/orgs/models.py:287]: we inherit capabilities from our parent.

File: temba/orgs/tests.py [hunk 4]
```diff
@@ -3856,30 +3857,6 @@ def test_import(self):
 
         self.login(self.admin)
 
-        # try importing without having purchased credits
```
Comment (nicpottier) [inline temba/orgs/tests.py:3859]: all orgs can import now

File: temba/orgs/models.py (hunk 3 above)
Comment (rowanseymour) [reply temba/orgs/models.py:287]: don't we need to set is_multi_org=False for child orgs? To prevent ending up with grandchildren orgs

Comment (nicpottier) [reply temba/orgs/models.py:287]: What's wrong with grandchildren orgs? In the IRC case they have a national office that will dole out credits to a regional office which will dole out credits to project workspaces.

Comment (rowanseymour) [reply temba/orgs/models.py:287]: Didn't know that was a thing we supported!

Comment (nicpottier) [reply temba/orgs/models.py:287]: Well it wasn't before, but ya, this is part of the reason I'm switching this.

**Comment by nicpottier** [comment]
@rowanseymour can we get this out this morning?

**Pull request merged**